platform/spire/src/command.py

```diff
@@ -53,6 +53,24 @@ def configure(self, command: list, parser: argparse.ArgumentParser):
             except AttributeError as e:
                 raise Exception("error configuring subcommand {!r}".format(subcommand)) from e
 
+        if 'help' not in self.mapping:
+            help_subparser = subparsers.add_parser(
+                'help',
+                description='print help about this command or a subcommnad',
+                help='show this help message and exit')
+            help_subparser.add_argument('subcommand', nargs='?')
+            def help_function(aargs):
+                if aargs.subcommand is None:
+                    parser.print_help()
+                else:
+                    try:
+                        subparsers.choices[aargs.subcommand].print_help()
+                    except KeyError:
+                        print('{}: no such subcommand'.format(aargs.subcommand))
+                        parser.print_help()
```

Comment:
I'm concerned about the possibility of print_help() raising a KeyError here. Is there any chance we could tweak this code so that only a missing `subparsers.choices[aargs.subcommand]` will trigger this error handling?